src/utils/prettifier.ts

```
@@ -0,0 +1,370 @@
+ */
+const handleClosingBracket = (
+    uglyString: string,
+    index: number,
+    state: PrettifyState
+): number => {
+    const closingBracket = uglyString[index];
+    if ("]" !== closingBracket) {
+        return index;
+    }
+
+    if ("[" === (state.regionStack[state.regionStack.length - 1] ?? {}).type) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)